boards.txt

```diff
@@ -2970,6 +2970,16 @@ GenF4.menu.pnum.BLACKPILL_F401CC.build.variant=STM32F4xx/F401CC(F-U-Y)_F401C(B-D
 GenF4.menu.pnum.BLACKPILL_F401CC.build.variant_h=variant_{build.board}.h
 GenF4.menu.pnum.BLACKPILL_F401CC.build.peripheral_pins=-DCUSTOM_PERIPHERAL_PINS
 
+# BlackPill F401CE
+GenF4.menu.pnum.BLACKPILL_F401CE=BlackPill F401CE
+GenF4.menu.pnum.BLACKPILL_F401CE.upload.maximum_size=524288
+GenF4.menu.pnum.BLACKPILL_F401CE.upload.maximum_data_size=98304
+GenF4.menu.pnum.BLACKPILL_F401CE.build.board=BLACKPILL_F401CE
+GenF4.menu.pnum.BLACKPILL_F401CE.build.product_line=STM32F401xE
+GenF4.menu.pnum.BLACKPILL_F401CE.build.variant=STM32F4xx/F401CC(F-U-Y)_F401C(B-D-E)(U-Y)
+GenF4.menu.pnum.BLACKPILL_F401CE.build.variant_h=variant_{build.board}.h
```

Comment:
To avoid duplicate files, I would rename the one existing like this `variant_BLACKPILL_F401Cx.h` and so:

```suggestion
GenF4.menu.pnum.BLACKPILL_F401CE.build.variant_h=variant_BLACKPILL_F401C.h
```

Same for the CC:
GenF4.menu.pnum.BLACKPILL_F401CC.build.variant_h=variant_{build.board}.h

Reply:
@alextrical 
I've made the changes and updated your branch. 
So pay attention to fetch your repo to update properly your branch.